### PR TITLE
{devel}[dummy/] Saxon-HE v9.7.0.4

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/s/Saxon-HE/Saxon-HE-9.7.0.4-Java-1.7.0_79.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+homepage = 'http://saxon.sourceforge.net'
+description = """Open Source SAXON XSLT processor developed by Saxonica Limited."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['SaxonHE%sJ.zip' % version.replace(".","-")]
+
+source_urls = ['https://sourceforge.net/projects/saxon/files/Saxon-HE/%(version_major_minor)s' , 'download']
+
+versionsuffix = '-Java-%(javaver)s'
```
Comment (boegel) [inline easybuild/easyconfigs/s/Saxon-HE/Saxon-HE-9.7.0.4-Java-1.7.0_79.eb]: @SimonPinches please move this up, right below `version =`

File: easybuild/easyconfigs/s/Saxon-HE/Saxon-HE-9.7.0.4-Java-1.7.0_79.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+name = 'Saxon-HE'
+version = '9.7.0.4'
+
+homepage = 'http://saxon.sourceforge.net'
+description = """Open Source SAXON XSLT processor developed by Saxonica Limited."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['SaxonHE%sJ.zip' % version.replace(".","-")]
+
```
Comment (boegel) [inline easybuild/easyconfigs/s/Saxon-HE/Saxon-HE-9.7.0.4-Java-1.7.0_79.eb]: please drop empty line here, we usually group `source_urls` and `sources`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2717.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2ab6759f58896fe34386755845ebc446 for a full test report.

**Comment by SimonPinches** [comment]
Updated as requested - apologies for the delay!

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/62b309588e8a41c2d7533d46dfdc2bd3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**